install-prebuilt-uefi/hikey-efi-prebuilt.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
This works for hikey960 only. It doesn't support original hikey (hi6220). So please change the file name to 'hikey960-efi-prebuilt.sh'